prereise/gather/demanddata/transportation_electrification/data_process.py

```diff
@@ -0,0 +1,133 @@
+import pandas as pd
+
+# Download CSV of NHTS 2017 from https://nhts.ornl.gov/
+
+
+def load_data(census_division: int, filepath: str = "trippub.csv"):
+    """Load the data at trippub.csv.
+
+    :param int census_division: any of the 9 census regions defined by the US census bureau
+    :param str filepath: filepath for the "trippub.csv" file in downloaded NHTS data
+    :raises ValueError: if the census division is not between 1 and 9, inclusive.
+    :return: (*pd.DataFrame*) -- the data loaded from trippub.csv.
```

Comment:
In docstrings and type annotations, we still want to use `pandas.DataFrame` rather than `pd.DataFrame`.